gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -93,30 +93,57 @@ def test_evaluation(self):
         assert_equal(actual, desired)
 
     def test_get_response(self):
-        # Get response
-        e_reco = EnergyBounds.equal_log_spacing(1 * u.GeV, 100 * u.TeV, 100)
-        response = self.edisp.get_response(1 * u.deg, 1 * u.TeV, e_reco)
-        actual = len(response)
-        desired = e_reco.nbins
-        assert_equal(actual, desired)
+        # Here we test get_response with an expected gaussian shape for edisp
+
+        # First we define a fake EnergyDispersion2D
```

Comment:
The generation of the fake edisp is very nice! However, it should not happen in the test but be available as method on ``EnergyDispersion2D``, similar to [EnergyDispersion.from_gauss](https://github.com/gammapy/gammapy/blob/master/gammapy/irf/energy_dispersion.py#L97) . The test should then just call this function.